client/selectors/index.js

```diff
@@ -6,6 +6,10 @@ import { ITEM_STATE } from '../constants'
 
 export const getAgendas = (state) => state.agenda.agendas
 export const getCurrentPlanningId = (state) => state.planning.currentPlanningId
+export const eventSearchActive = (state) => (typeof state.events.search !== 'undefined') ? ((
+    state.events.search.currentSearch.advancedSearch ||
+    state.events.search.currentSearch.fulltext) ?
+    true : false) : false
```

Comment:
this causes test failure at - client/components/EventsListContainer/_test.jsx:10:24 and lient/components/tests/PlanningApp_test.jsx:20:24 for "TypeError: state.events.search.currentSearch is undefined in tests.js (line 84899)", but I have not been able to resolve the issue.  Ive tried setting it in initialState passed to createTestStore, and also setting a default in utils/index.js but nothing seems to work.  Not sure how to resolve the issue.